x-pack/plugins/security_solution/public/common/components/event_details/summary_view.tsx

```diff
@@ -77,118 +89,212 @@ const StyledEuiInMemoryTable = styled(EuiInMemoryTable as any)`
   eventId,
   fieldName,
-  value,
+  values,
   fieldType = '',
   linkValue,
-}: SummaryRow['description']) => (
-  <FormattedFieldValue
-    contextId={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}`}
-    eventId={eventId}
-    fieldName={fieldName}
-    fieldType={fieldType}
-    value={value}
-    linkValue={linkValue}
-  />
-);
+  isDraggingDisabled,
+  isRoleSeparator,
+}: SummaryRow['description']) =>
+  values.map((value) => (
+    <FormattedFieldValue
+      key={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}-key`}
+      contextId={`alert-details-value-formatted-field-value-${contextId}-${eventId}-${fieldName}-${value}`}
+      eventId={eventId}
+      fieldName={fieldName}
+      fieldType={fieldType}
+      value={value}
+      linkValue={linkValue}
+      isDraggingDisabled={isDraggingDisabled}
+      isRoleSeparator={isRoleSeparator}
+    />
+  ));
 
 const getSummary = ({
   data,
   browserFields,
-  timelineId,
+  timelineId: contextId,
   eventId,
+  isDisplayingThreatSummary,
+  isDisplayingThreatDetails,
 }: {
   data: TimelineEventsDetailsItem[];
-  browserFields: BrowserFields;
+  browserFields?: BrowserFields;
   timelineId: string;
   eventId: string;
+  isDisplayingThreatSummary: boolean;
+  isDisplayingThreatDetails: boolean;
 }) => {
-  return data != null
-    ? fields.reduce<Summary>((acc, item) => {
-        const field = data.find((d) => d.field === item.id);
-        if (!field) {
-          return acc;
-        }
-        const linkValueField =
-          item.linkField != null && data.find((d) => d.field === item.linkField);
-        const linkValue = getOr(null, 'originalValue.0', linkValueField);
-        const value = getOr(null, 'originalValue.0', field);
-        const category = field.category;
-        const fieldType = get(`${category}.fields.${field.field}.type`, browserFields) as string;
-        const description = {
-          contextId: timelineId,
-          eventId,
-          fieldName: item.id,
-          value,
-          fieldType: item.fieldType ?? fieldType,
-          linkValue: linkValue ?? undefined,
-        };
-
-        if (item.id === 'signal.threshold_result.terms') {
-          try {
-            const terms = getOr(null, 'originalValue', field);
-            const parsedValue = terms.map((term: string) => JSON.parse(term));
-            const thresholdTerms = (parsedValue ?? []).map(
-              (entry: { field: string; value: string }) => {
-                return {
-                  title: `${entry.field} [threshold]`,
-                  description: {
-                    ...description,
-                    value: entry.value,
-                  },
-                };
-              }
-            );
-            return [...acc, ...thresholdTerms];
-          } catch (err) {
-            return acc;
+  if (!data) return [];
+  if (isDisplayingThreatSummary || isDisplayingThreatDetails) {
```

Comment:
After looking through these changes, my opinion is that the cost to add these new tabs to the existing SummaryView component is not worth whatever benefit it might provide.

I think the best example is this `if/else` logic, where one branch is the original SummaryView logic, and the other is our new logic for CTI. That strongly implies that these should be separated.

At a _very_  high level, I'm thinking something like this:

```jsx
<SummaryTab>
  <SummaryView />
  <CTITabbedContent>
    <CTISummary />
    <CTIDetails />
  </CTITabbedContent>
</SummaryTab>
```

If there is _presentational_ logic that we can extract from `SummaryView` to be used by `SummaryView`, `CTISummary`, and `CTIDetails`, let's absolutely do that, but it feels like we're bending over backwards here to try to make it all work within `SummaryView`.

I'm sure that you've thought about this more than I have, so I'd love to pair on this to see if we can improve it together!